packages/foundation/tools/eslint-config/index.js

```diff
@@ -186,6 +187,26 @@ export default tseslint.config(
       'jest/valid-title': 'off',
     },
   },
+  {
+    files: ['**/*.spec.ts'],
+    plugins: {
+      vitest,
+    },
+    rules: {
+      ...vitest.configs.recommended.rules,
+      ...vitest.configs.all.rules,
+      '@typescript-eslint/unbound-method': 'off',
+      '@typescript-eslint/no-confusing-void-expression': 'off',
+      '@typescript-eslint/no-magic-numbers': 'off',
+      '@typescript-eslint/no-unsafe-assignment': 'off',
+      'vitest/consistent-test-filename': 'off',
+      'vitest/max-expects': 'off',
+      'vitest/no-hooks': 'off',
+      'vitest/prefer-expect-assertions': 'off',
+      'vitest/prefer-strict-equal': 'error',
+      'vitest/valid-title': 'off',
+    },
+  },
```

Comment:
I think these rules are causing trouble to non migrated test files. Should we migrate everything in this PR?